src/App.tsx

```diff
@@ -43,6 +45,26 @@ const router = createBrowserRouter([
 	},
 ]);
 
-const App = () => <RouterProvider router={router} />;
+const App = () => {
+	const { oidcUser } = useOidcUser();
+	const emptyUser = {
+		id: 0,
+		authenticationId: '',
+		createdAt: new Date(0),
+	};
+	const [currentUserState, setCurrentUserState] = useState(emptyUser);
```

Comment:
I would rather `currentUserState` be potentially `undefined` than have a placeholder empty user.  Ultimately a user may or may not exist, and using the built in `undefined` instead of a specialized empty user object will make it easier to determine that state down the line.

Reply:
Good point!